src/app.py

```diff
@@ -2982,6 +3072,18 @@ def get_prov_info_for_dataset(id):
             internal_dict[HEADER_PROCESSED_DATASET_UUID] = ",".join(processed_dataset_status_list)
             internal_dict[HEADER_PROCESSED_DATASET_UUID] = ",".join(processed_dataset_portal_url_list)
 
+    if include_samples:
+        dataset_samples = app_neo4j_queries.get_all_dataset_samples(neo4j_driver_instance, uuid)
+        logger.debug(f"dataset_samples={str(dataset_samples)}")
+        if 'all' in include_samples:
+            internal_dict[HEADER_DATASET_SAMPLES] = dataset_samples
```

Comment:
Not to disparage the fine people at GitHub, but can you confirm this is correctly attributed?  I don't see it at https://github.com/sennetconsortium/entity-api/compare/main...karlburke/collectionsAsEntities#diff-04791d82dd15fdd480f084d7ef65a10789fa5012cb7935f76080763444d48a00L2980-L3002
and it isn't anything I deliberately worked on.

I see it as a difference when I look at "Files Changed" on the PR, but Joe has told me this is some kind of "3 dot diff".  Perhaps it is useful for showing off at conferences I do not attend, but otherwise it seems like one more shortcoming of Git relative to direct source control, to a cranky old man like me!